index.js

```diff
@@ -338,18 +362,21 @@ function handleRequests() {
 /**
  * @type { typeof import('./index').init }
  */
-function init(aApp, aPredefinedSpec = {}, aSpecOutputPath = undefined, aWriteInterval = 1000 * 10, aSwaggerUiServePath = 'api-docs', mongooseModels = []) {
+function init(aApp, aPredefinedSpec = {}, aSpecOutputPath = undefined, aWriteInterval = 1000 * 10, aSwaggerUiServePath = 'api-docs', aMongooseModels = [], aTags = undefined) {
   handleResponses(aApp, {
     swaggerUiServePath: aSwaggerUiServePath,
     specOutputPath: aSpecOutputPath,
     predefinedSpec: aPredefinedSpec,
     writeIntervalMs: aWriteInterval,
+    mongooseModels: aMongooseModels,
+    tags: aTags
   });
   setTimeout(() => {
     handleRequests();
   }, 1000);
   
-  updateDefinitionsSpec(mongooseModels);
+  updateDefinitionsSpec(aMongooseModels);
+  updateTagsSpec(aTags || aMongooseModels);
```

Comment:
actually two lines above are not needed because `handleResponses()` does it already, I'll fix it

Reply:
https://github.com/mpashkovskiy/express-oas-generator/commit/33f841707443874c7ba08e9a668d8f73deda1db1

Reply:
You're right 👍